provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/endpoint/EndpointTransformer.java

```diff
@@ -1,3 +1,51 @@
 package gov.va.api.health.providerdirectory.service.controller.endpoint;
 
-public class EndpointTransformer {}
+import gov.va.api.health.providerdirectory.service.AddressResponse;
+import gov.va.api.health.stu3.api.datatypes.CodeableConcept;
+import gov.va.api.health.stu3.api.datatypes.Coding;
+import gov.va.api.health.stu3.api.resources.Endpoint;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
+public class EndpointTransformer implements EndpointController.Transformer {
+
+  /* managingOrganization will map to 'facility' field from VLER.
+   * connectionType and payloadType are defaults for VLER. */
+  @Override
+  public Endpoint apply(EndpointWrapper vlerData) {
+    AddressResponse.Contacts response = vlerData.addressResponse().contacts().get(0);
+    return Endpoint.builder()
+        .resourceType("Endpoint")
+        .status(status("active"))
+        .connectionType(connectionCodeCodings("direct-project"))
+        .name(response.displayname())
+        .payloadType(payloadCodeCodings("VLER Direct"))
+        .address(response.mail())
+        .build();
+  }
+
+  Coding connectionCodeCodings(String connectionTypeResponse) {
+    if (connectionTypeResponse == null) {
+      return null;
+    }
```

Comment:
Similar to previous comment... if this method is only ever invoked with `direct-project` as the argument, then you don't need the null check. You don't really need the method either; just inline `Coding.builder()....` where it is used.